r/R/util.R

```diff
@@ -143,8 +143,8 @@ handle_parquet_io_error <- function(e, format, call) {
       msg,
       i = "Did you mean to specify a 'format' other than the default (parquet)?"
     )
+    abort(msg, call = call)
```

Comment:
Why these changes?

Reply:
Oh I see, so that you could chain together multiple of these. I suspect we could have a better developer experience, this change makes it easy to accidentally swallow all other errors. Could you make a followup jira for this, and leave a big comment above these?

Reply:
Sorry, I don't understand, a followup JIRA to do what?

Reply:
Have created [ARROW-17355](https://issues.apache.org/jira/browse/ARROW-17355) and added a comment now.